build/build-version.js

```diff
@@ -0,0 +1,11 @@
+'use strict'
+
```

Comment:
Do we already have tests that depend on version being there that would be failing if this won't work?

Reply:
yes: https://github.com/fastify/fastify/blob/master/test/plugin.test.js#L821